trace/cloud-trace-demo-app-opentelemetry/README.md

```diff
@@ -2,73 +2,107 @@
 
-#### Demo Requirements
+## Demo Requirements
+
 If you are using Cloud Shell, skip to the next section.
 
 1. Install gcloud <https://cloud.google.com/sdk/install>
 2. Install kubectl <https://kubernetes.io/docs/tasks/tools/install-kubectl/>
 3. Install docker <https://docs.docker.com/install/>
 
-    
-#### Create a GKE cluster
-
-4. Enable Google Cloud and set up region and zone.
-
-    `gcloud init`
-5. Enable the GKE API & billing:
-
-    `gcloud services enable container.googleapis.com`
-6. Create a GKE cluster named "cloud-trace-demo", replacing `your-gcp-zone`  below
-with the 
-[GCP Zone](https://cloud.google.com/compute/docs/regions-zones) closest in proximity to you:
-
-     ```
-     gcloud container clusters create cloud-trace-demo\`
-     --num-nodes 1 \
-     --enable-basic-auth \
-     --issue-client-certificate \
-     --zone your-gcp-zone
-     ```
-7. Verify that you have access to the cluster:
-    
-    `kubectl get nodes`
-
-#### Deploy The Cloud Trace Demo App
-
-8. Build and tag the docker image for demo app:
-
-    `docker build -t gcr.io/${PROJECT_ID}/cloud-trace-demo:v1 .`
-9. Deploy resource to the cluster:
-
-    `kubectl apply -f deployment.yaml`
-10. Track the status of the deployment:
-
-    `kubectl get deployments`
-    
-    Deployment is complete when all of the available deployments are ready. 
-11. Run the following command to see the pods the deployment created:
-    
-    `kubectl get pods`  
-
-#### Deploy The Cloud Trace Demo Service
-
-12. Create the cloud trace demo service:
-    
-    `kubectl apply -f service.yaml`
-13. Get the services IP address by running the following command:
-    
-    `kubectl get services`
-14. Send a curl request to the EXTERNAL_IP, replacing `EXTERNAL_IP` with the external IP address found
-in step 13:
-    
-    `curl EXTERNAL_IP`
-15. Visit [Trace List](https://console.cloud.google.com/traces/list) to check traces generated.
+## Deploy and run the demo application
+
+1. Enable Google Cloud and set up region and zone.
+
+   ```bash
+   gcloud init
+   ```
+
+1. Enable the GKE API:
+
+   ```bash
+   gcloud services enable container.googleapis.com
+   ```
+
+1. Setup GCP project to `YOUR_PROJECT_ID`. Please, replace `YOUR_PROJECT_ID` with your GCP project id.
+
+   ```bash
+   gcloud config set project YOUR_PROJECT_ID
+   ```
+
+1. Setup the location of GKE cluster and create a new cluster named
+"cloud-trace-demo".
+The example below sets the cluster's location to the "us-central1-c" zone.
+You can replace the zone with one which you like. See [zones][] for the full
+list.
+
+   ```bash
+   ZONE=us-central1-c
+   gcloud container clusters create cloud-trace-demo \
+       --zone $ZONE
+   ```
+
+1. Update GKE cluster credentials and verify the access to the cluster:
+
+   ```bash
+   gcloud container clusters get-credentials cloud-trace-demo --zone $ZONE
+   kubectl get nodes
+   ```
+
+   The output is expected to display 3 work nodes.
```

Comment:
Any examples for these 3 work nodes?

Reply:
The names of the nodes will be different each time due to auto-generated process. I think that it is sufficient to say that there will be a list of three records (or lines).

Reply:
If it's consistent output with just different names, something like the example on https://cloud.google.com/kubernetes-engine/docs/tutorials/stateful-workload#configure_the_managed_file_storage_with_using_csi helps quite a lot for users to help verify. Take a look at the example output with the text `The output is similar to the following:`.

Reply:
Thank you for the example.